comp/core/flare/helpers/builder.go

```diff
@@ -317,13 +364,21 @@ func (fb *builder) PrepareFilePath(path string) (string, error) {
 }
 
 func (fb *builder) RegisterFilePerm(path string) {
+	fb.Lock()
+	defer fb.Unlock()
```

Comment:
should we check if the flare is closed and permsInfos file has been closed too ?

Reply:
`permsInfos` is just a map so there is no real issue in adding a value after the flare is already closed, it just won't be read
But I'll handle that case explicitly to make it clear 👍